SPECS/compiler-rt/compiler-rt.spec

```diff
@@ -0,0 +1,72 @@
+%global maj_ver 12
+%global min_ver 0
+%global patch_ver 1
+
+%global compiler_rt_srcdir %{name}-%{version}.src
```

Comment:
Is "_srcdir" the right suffix for this macro?  Seems like this is actually the "source" name since it's used in "Source0"